src/styles/home.module.css

```diff
@@ -19,76 +19,137 @@
   margin-bottom: 20px;
 }
 
-.btn {
-  display: inline-block;
-  background: blue;
-  padding: 10px 16px;
-  border-radius: 10px;
-  margin-top: 20px;
-  font-weight: 500;
+.episodes {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 24px 0;
+  background: var(--grey-100);
 }
 
-.postList {
-  border-radius: 4px;
-  border: 0.5px solid #dcdcdc;
-  background-color: white; /*royalblue;*/
-  margin: 40px 0px;
-  display: grid;
-  grid-template-columns: 1fr;
-  grid-gap: 0px;
-  box-shadow: 0px 3px 15px 0px rgb(0 0 0 / 15%);
+.episodes a:hover {
+  text-decoration: none;
```

Comment:
We need this to make sure that card text is not underlined when a card is hovered.